src/components/AlumniTimeline.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from './AlumniTimeline.module.css';
+import Members from "./Members";
+
+const AlumniOfClass = ({classOf, alumni}) => {
+  return (
+    <li className={styles.event}>
+      <label className={styles.icon}></label>
+      <div className={styles.body}>
+        <p className={styles.date}>Class of {classOf}</p>
+          <Members members={alumni} />
+      </div>
+    </li>
+  )
+}
+
+const AlumniTimeline = ({alumni}) => {
```

Comment:
This file is effectively JavaScript under the name TypeScript (since I'm not so familiar with it). Will specify the types later.

Also, the logic in this function is incredibly ugly... @SamChou19815 do you have any suggestions or ideas to make it more elegant?